src/Employer/Employer.Web/Orchestrators/Part1/TitleOrchestrator.cs

```diff
@@ -53,13 +60,13 @@ public TitleViewModel GetTitleViewModel()
             return vm;
         }
 
-        public async Task<Guid> PostTitleEditModelAsync(TitleEditModel vm)
+        public async Task<OrchestratorResponse<Guid>> PostTitleEditModelAsync(TitleEditModel vm)
         {
             if (!vm.VacancyId.HasValue)
             {
                 var id = await _client.CreateVacancyAsync(vm.Title, vm.EmployerAccountId);
 
-                return id;
+                return new OrchestratorResponse<Guid>(id);;
```

Comment:
The Title will also need validating when the vacancy is created.

Reply:
Yep, just not fully implemented yet. Focussed only on the Update.